### PR TITLE
Update RSA10a for 0.9

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/Auth.swift [hunk 1]
```diff
@@ -1617,7 +1617,7 @@ class Auth : QuickSpec {
         describe("authorise") {
 
             // RSA10a
-            it("should create a token if needed and use it") {
+            it("should reuse the valid token and ensure Token Auth is used for all future requests") {
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: Nope, it should not reused the valid token, but should create a token and ensure Token Auth...

Comment (ricardopereira) [inline Spec/Auth.swift]: Fixed ed52009.

**Comment by ricardopereira** [comment]
If the spec looks good then I'll fix the client.

**Comment by tcard** [comment]
The spec LGTM.

**Pull request merged**